txhelpers/txhelpers.go

```diff
@@ -584,3 +584,29 @@ func DetermineTxTypeString(msgTx *wire.MsgTx) string {
 		return "Regular"
 	}
 }
+
+// TxFee computes and returns the fee for a given tx
+func TxFee(msgTx *wire.MsgTx) float64 {
+	var amtIn int64
+	for iv := range msgTx.TxIn {
+		amtIn += msgTx.TxIn[iv].ValueIn
+	}
+	var amtOut int64
+	for iv := range msgTx.TxOut {
+		amtOut += msgTx.TxOut[iv].Value
+	}
+	return dcrutil.Amount(amtIn - amtOut).ToCoin()
+}
+
+// TxFeeRates computes and returns the fee rate in DCR/KB for a given tx
+func TxFeeRate(msgTx *wire.MsgTx) float64 {
```

Comment:
This could have a second output to give absolute fee too.  This would avoid the need to call TxFee after this.